agents/mox_kmd_ee/tests/common.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import os
+import json
+import sys
+import pdb
+import functools
+import traceback
+
+
+def debug(*exceptions):
+    if not exceptions:
+        exceptions = (AssertionError, )
+    debug_on_error = os.environ.get("TESTPMPDB", False)
+
+    def decorator(f):
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            try:
+                return f(*args, **kwargs)
+            except exceptions:
+                if not debug_on_error:
+                    raise
+                info = sys.exc_info()
+                traceback.print_exception(*info)
+                pdb.post_mortem(info[2])
+        return wrapper
+    return decorator
+
+
+class Test(unittest.TestCase):
+
+    # @debug(UnicodeDecodeError, json.decoder.JSONDecodeError)
```

Comment:
Virker dekoratoren ikke endnu, eller hvordan?

Reply:
Dekoratoren er praktisk til at kaste testen ind i debug ved bestemte fejltyper for at 'prøve sig frem' - vi kunne godt fjerne den eller give den en lille kommentar med på vejen, men den skal ikke være på til 'hverdag'- der skal testene jo bare gå i dørken.

Reply:
Den har fået en bemærkning